lib/Api/BookingsApi.php

```diff
@@ -0,0 +1,2628 @@
+<?php
+/**
+ * BookingsApi
+ * PHP version 7.4
+ *
+ * @category Class
+ * @package  Katanox
+ * @author   OpenAPI Generator team
```

Comment:
is it possible to change the author?

Reply:
Apparently it is hardcoded and cannot be modified: https://github.com/OpenAPITools/openapi-generator/issues/7530#issuecomment-712962511